docs/protocol.rst

```diff
@@ -44,13 +44,13 @@ payload.  All data transmitted via |MHP| are split into chunks.
 
 **Channel** refers to a set of chunks transmitted over the same connection,
 identified among other ones using a number that is unique throughout the
-currently active channels connection-wise.  Channels provide multiplexing
+currently active channels connection-wise.  Channel provides multiplexing
 capabilities for connections.
 
 **Message** refers to a channel that is characterized by a short lifetime and
-small number of chunks (only one chunk in most cases, with the maximal amount
+small number of chunks (only one chunk in most cases, with the maximal length
```

Comment:
The word in question refers to the count of chunks, so I don't feel like "length" is appropriate here.